bower.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "adapt-navigationTitle",
   "version": "1.0.0",
-  "framework": ">=5",
+  "framework": ">=5.20.0",
```

Comment:
With the removal of `.u-float-left` from the less this should be bumped to 5.24.4

Reply:
Done.